runtime/bcverify/staticverify.c

```diff
@@ -1307,32 +1308,28 @@ checkStackMap (J9CfrClassFile* classfile, J9CfrMethod * method, J9CfrAttributeCo
 					slotCount = 0;
 					if ((frameType >= CFR_STACKMAP_SAME_LOCALS_1_STACK) && (frameType <= CFR_STACKMAP_SAME_LOCALS_1_STACK_EXTENDED)) {
 						slotCount = 1;
-
```

Comment:
@ChengJin01 I double checked other frame types that may be affected by this change. `same_locals_1_stack_item_frame` and `same_locals_1_stack_item_frame_extended` will go through the loop. They should not cause any unexpected failures.

I've removed this check since the new maxStack verification in `checkStackMapEntries` will cover this.

Reply:
I'm OK with the latest changes.